_rules/aria-hidden-no-focusable-content-6cfa84.md

```diff
@@ -97,13 +97,28 @@ This `input` element is not part of the [sequential focus navigation][] because
 
 #### Passed Example 4
 
-This `a` element is part of the [sequential focus navigation][], but moves focus to the `input` element whenever it receives focus.
+This `a` element is not [focusable][] because it moves focus to the `input` element whenever it receives focus. These elements
```

Comment:
This example (and the next one) does not look like it is related to ARIA 1.2 (nor to the rule that the PR mentions in its title).
Should it be moved to another PR?
(or maybe it's because something has been merged but the `wai-aria-1.2` branch hasn't been updated to the latest `develop` so git get confused)